include/RE/I/INISettingCollection.h

```diff
@@ -19,14 +20,28 @@ namespace RE
 			return *singleton;
 		}
 
-		[[nodiscard]] Setting* GetSetting(const std::string_view a_name)
+		std::string ToLowerStrV(std::string_view _sourceStringView)
```

Comment:
ideally this should be moved somewhere else, but for now it can just be made private as it's an implementation detail.